README.md

```diff
@@ -169,82 +188,90 @@ In the following section we guide you trough installing and running a Gazebo sim
    make px4_sitl_default gazebo
+   
+   # Quit the simulation (Ctrl+C)
 
-   # Setup some more Gazebo-related environment variables (You may need to modify this line based on the location of the Firmware folder on your machine)
+   # Setup some more Gazebo-related environment variables (modify this line based on the location of the Firmware folder on your machine)
    . ~/Firmware/Tools/setup_gazebo.bash ~/Firmware ~/Firmware/build/px4_sitl_default
    ```
 
-1. Add the Firmware directory to ROS_PACKAGE_PATH so that ROS can start PX4.
-
+1. Add the Firmware directory to ROS_PACKAGE_PATH so that ROS can start PX4:
    ```bash
    export ROS_PACKAGE_PATH=${ROS_PACKAGE_PATH}:~/Firmware
    ```
 1. Finally, set the GAZEBO_MODEL_PATH in your bashrc:
-```bash
-echo export GAZEBO_MODEL_PATH=${GAZEBO_MODEL_PATH}:~/catkin_ws/src/avoidance/avoidance/sim/models:~/catkin_ws/src/avoidance/avoidance/sim/worlds >> ~/.bashrc
-```
+   ```bash
+   echo export GAZEBO_MODEL_PATH=${GAZEBO_MODEL_PATH}:~/catkin_ws/src/avoidance/avoidance/sim/models:~/catkin_ws/src/avoidance/avoidance/sim/worlds >> ~/.bashrc
+   ```
 
-Steps 3, 4 and 5 together with sourcing your catkin setup.bash (`source ~/catkin_ws/devel/setup.bash`) should be repeated each time a new terminal window is open.
+The last three steps, together with sourcing your catkin **setup.bash** (`source ~/catkin_ws/devel/setup.bash`) should be repeated each time a new terminal window is open.
 You should now be ready to run the simulation using local or global planner.
 
 ### Local Planner (default, heavily flight tested)
 
 This section shows how to start the *local_planner* and use it for avoidance in mission or offboard mode.
 
-The planner is based on the [3DVFH+](http://ceur-ws.org/Vol-1319/morse14_paper_08.pdf) algorithm. To run the algorithm it is possible to
+The planner is based on the [3DVFH+](http://ceur-ws.org/Vol-1319/morse14_paper_08.pdf) algorithm.
 
-* simulate a forward looking stereo camera running OpenCV's block matching algorithm (SGBM by default)
-
-   ```bash
-   # if stereo-image-proc not yet installed
-   sudo apt install ros-kinetic-stereo-image-proc
-
-   roslaunch local_planner local_planner_stereo.launch
-   ```
-
-   The disparity map from `stereo-image-proc` is published as a [stereo_msgs/DisparityImage](http://docs.ros.org/api/stereo_msgs/html/msg/DisparityImage.html) message, which is not supported by rviz or rqt. To visualize the    message, either run:
-
-   ```bash
-   # if image_view is not yet installed
-   sudo apt install ros-kinetic-image-view
-
-   rosrun image_view stereo_view stereo:=/stereo image:=image_rect_color
-   ```
+You *may* need to install some additional dependencies (if not already installed):
+* Melodic:
+  ```sh
+  sudo apt install ros-melodic-stereo-image-proc ros-melodic-image-view
+  ```
+* Kinetic:
+  ```sh
+  sudo apt install ros-kinetic-stereo-image-proc ros-kinetic-image-view
+  ```
+   
+To run the algorithm it is possible to:
```

Comment:
Do you really mean "you can do this" or are you asking the person to do it?  Should they do it in the same terminal? If they do it in this terminal, I assume they do have to set up all the environment stuff again?

Reply:
Actually, what I think you're saying is that you can simulate depth camera for object avoidance OR a single camera or three cameras - for collision prevention. Correct?

Presumably that means you can only do either of object avoidance / collision prevention testing at the one time?

Reply:
To run the local planner you can use 3 different scripts:
- local_planner_depth_camera : vehicle has one forward facing kinect sensor 
-  local_planner_sitl_3cam : vehicle has 3 kinect sensors (left, right, front)
- local_planner_stereo: vehicle has a stereo camera and openCV block matching is used to generate depth information

With all three launch file you can run Obstacle Avoidance and Collision Prevention. The algorithm running is the same, ony the sensors on the vehicle differ.

Reply:
Thanks, very helpful.